bucket/nvidia-display-driver-dch-np.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "471.41",
+    "version": "528.24",
     "description": "NVIDIA display driver.",
```

Comment:
Can we add a suffix that describes what each file is for?  So
```json
"description": "NVIDIA display driver (Declarative Componentized Hardware version).",
```
and I guess add this to the other file:
```json
"description": "NVIDIA display driver (Standard version).",
```

Reply:
SGTM. Also, my script has both packages using the same version! I'll make a new comment for that

Reply:
The distinction between "DCH" and "standard" is something I found quite confusing. However, IMO most users should be installing DCH drivers

![image](https://user-images.githubusercontent.com/1100550/215371204-99cc8a45-d935-4ffc-a3a2-247a4129c8c2.png)

I added a bit of context from NVIDIA to each driver description. I've baked in a bit of pro-DCH bias, but I quoted directly from NVIDIA's descriptions